runtime/dart_isolate.cc

```diff
@@ -1117,14 +1276,25 @@ void DartIsolate::OnShutdownCallback() {
   }
 }
 
+void DartIsolate::OnMessageEpilogue(Dart_Handle result) {
+  if (is_platform_isolate_ && Dart_CurrentIsolate() != nullptr) {
+    FML_DCHECK(Dart_CurrentIsolate() == isolate());
+    FML_DCHECK(platform_isolate_pending_messages_ > 0);
+    --platform_isolate_pending_messages_;
+    if (platform_isolate_pending_messages_ == 0 && !Dart_HasLivePorts()) {
```

Comment:
We do want to keep the platform isolate alive - even if there's no pending messages.

Could you add a comment here explaining more what is happening here? In which circumstance does it not have life ports anymore?